brainiak/fcma/util.py

```diff
@@ -115,4 +121,9 @@ def compute_correlation(matrix1, matrix2):
                                               0.0,
                                               corr_data,
                                               r2)
+
+    # optionally convert zeros back to NaNs
+    if return_nans:
+        corr_data[corr_data == 0] = np.nan
```

Comment:
It is possible that we have a pair whose correlation is 0. You may want to add this check to line 50, in `_normalize_for_correlation`

Reply:
Good point—makes sense. New commit seems to behave as expected.